src/components/ButtonWithDropdownMenu/index.tsx

```diff
@@ -97,7 +98,7 @@ function ButtonWithDropdownMenu<IValueType>({
     return (
         <View style={wrapperStyle}>
             {shouldAlwaysShowDropdownMenu || options.length > 1 ? (
-                <View style={[styles.flexRow, styles.justifyContentBetween, styles.alignItemsCenter, style]}>
```

Comment:
Looks like a breaking change.. Are you sure it doesn't break anything in other parts of the app? 👀

Reply:
You were right, I've missed one place, right now it is fine everywhere, because we need this styling only when `isSplitButton` is set to true